internal/providers/terraform/azure/testdata/cognitive_account_test/cognitive_account_test.golden

```diff
@@ -178,9 +178,9 @@
  └─ Text analytics for health (5K-500K)                               Monthly cost depends on usage: $25.00 per 1K records         
                                                                                                                                    
  azurerm_cognitive_account.with_usage["SpeechServices-S0"]                                                                         
- ├─ Speech to text                                                                    20  hours                           $20.00   
+ ├─ Speech to text                                                                    20  hours                        not found   
```

Comment:
We should create a follow ticket to investigate why these are "not found" if you haven't already done so, just so we don't lose track of it.

Reply:
added to this ticket - https://www.notion.so/infracost/Different-Plans-5fd3a5e4fddc414690cdb7944d07001e?pvs=4

Thanks for review